compute/resource_cluster.go

```diff
@@ -101,6 +101,11 @@ func resourceClusterSchema() map[string]*schema.Schema {
 			p.Sensitive = true
 		}
 		s["autotermination_minutes"].Default = 60
+		s["num_workers"] = &schema.Schema{
+			Type:     schema.TypeInt,
+			Optional: true,
```

Comment:
what do acceptance tests say about this change?

Reply:
I've added a test, is it enough?

Reply:
@sabino no, it's not enough :)

1. please run `make test-azcli` and attach the output of `scripts/azcli-integration/test-report.log`. You're on Azure, right?
2. please modify documentation for cluster resource

I a have strange hunch that this might break some functionality and want to have it properly tested before merging.

Reply:
I'm actually running on AWS - Single Tenant.

Anyway, I'll update the documentation.
Btw, thanks for the follow up!

Reply:
I'll create a trial account and run the tests.

Reply:
@nfx FYI looks like it is not possible to use a trial account, which is a bummer. I've tried Pay-As-You-Go as well and they still have limits for those. I've requested a quota increase, but I'm not sure how long it will take.

![image](https://user-images.githubusercontent.com/982190/96941945-17218280-14aa-11eb-82bf-6e1e91c9e398.png)

Reply:
@sabino, just in case, i'm interested in output of only `TestAcc*Cluster*` tests. They are independent of a cloud. See contributing docs for guidelines how to run tests

Reply:
@nfx is this it?

I've used this command:
`DATABRICKS_CONFIG_PROFILE="staging" ./run.sh awsst "TestAcc*Cluster*" --debug --tee`

- `test-report.log`

```
awsst
---

 * [x] compute/acceptance.TestAccClusterPolicyResourceFullLifecycle (8.14s)
 * [x] compute/acceptance.TestAccClusterResource_CreateClusterWithLibraries (158.32s)
```

- `out.log`
```
EMPTY .
testing: warning: no tests to run
coverage: 0.0% of statements
PASS common
testing: warning: no tests to run
coverage: 16.3% of statements
PASS internal
testing: warning: no tests to run
coverage: 0.0% of statements
PASS identity
testing: warning: no tests to run
coverage: 0.0% of statements
PASS internal/qa
testing: warning: no tests to run
coverage: 0.0% of statements
PASS internal/util
testing: warning: no tests to run
coverage: [no statements]
PASS access/acceptance
testing: warning: no tests to run
coverage: 5.8% of statements
PASS compute
testing: warning: no tests to run
coverage: 0.0% of statements
PASS access
testing: warning: no tests to run
coverage: [no statements]
PASS identity/acceptance
testing: warning: no tests to run
coverage: 0.0% of statements
PASS internal/sanity
testing: warning: no tests to run
coverage: 0.8% of statements
PASS mws
testing: warning: no tests to run
coverage: [no statements]
PASS mws/acceptance
testing: warning: no tests to run
coverage: 0.0% of statements
PASS provider
testing: warning: no tests to run
coverage: 0.0% of statements
PASS storage
testing: warning: no tests to run
coverage: [no statements]
PASS storage/acceptance
testing: warning: no tests to run
coverage: 0.0% of statements
PASS workspace
testing: warning: no tests to run
coverage: [no statements]
PASS workspace/acceptance
PASS compute/acceptance.TestAccClusterPolicyResourceFullLifecycle (8.14s)
PASS compute/acceptance.TestAccClusterResource_CreateClusterWithLibraries (158.32s)
coverage: [no statements]
PASS compute/acceptance
EMPTY internal/acceptance

DONE 2 tests in 170.845s
```